@kiva/kv-components/src/vue/KvLendCta.vue

```diff
@@ -70,7 +70,7 @@
 				<!-- MOBILE-ONLY: Content for multiple preset buttons -->
 				<div
 					v-if="showPresetAmounts && !isAdding && !isLendAmountButton &&
-						presetButtonsPrices.length > 1 && !isNarrowDesktop"
+						presetButtonsPrices.length > 1 && !isNarrowDesktop && viewportWidth < 640"
```

Comment:
Why was 640 picked? Also, it's still happening for larger mobile screen sizes:

<img width="432" height="293" alt="Image" src="https://github.com/user-attachments/assets/9bb24b19-155b-4090-b2a6-6780fb2458d2" />

Reply:
Just seeing this! I thought the value just had to be less than the iPad tablet size but now pushing up commit that uses defined mobile breakpoint already in file!